---
layout: page
title: "FHIR Chat  · ValueSet error running IG publisher · committers"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179165-committers/index.html">committers</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179165-committers/topic/ValueSet.20error.20running.20IG.20publisher.html">ValueSet error running IG publisher</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="153850322"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179165-committers/topic/ValueSet%20error%20running%20IG%20publisher/near/153850322" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Sean McIlvenna <a href="https://fhir.github.io/chat-archive/stream/179165-committers/topic/ValueSet.20error.20running.20IG.20publisher.html#153850322">(Sep 22 2016 at 05:27)</a>:</h4>
<p>I'm getting this odd error from the IG publisher when I include a ValueSet in the resources directory (didn't even reference it in the ImplementationGuide or control file):</p>
<p>Exception in thread "main" java.lang.Error: fetching org.hl7.fhir.dstu3.model.ValueSet not done yet for URI urn:oid:2.16.840.1.113883.1.11.1<br>
at org.hl7.fhir.dstu3.utils.SimpleWorkerContext.fetchResource(SimpleWorkerContext.java:343)<br>
at org.hl7.fhir.igtools.publisher.IGKnowledgeProvider.resolveBinding(IGKnowledgeProvider.java:331)<br>
at org.hl7.fhir.dstu3.utils.ProfileUtilities.generateDescription(ProfileUtilities.java:1869)<br>
at org.hl7.fhir.dstu3.utils.ProfileUtilities.generateDescription(ProfileUtilities.java:1804)<br>
at org.hl7.fhir.dstu3.utils.ProfileUtilities.genElement(ProfileUtilities.java:1676)<br>
at org.hl7.fhir.dstu3.utils.ProfileUtilities.genElement(ProfileUtilities.java:1702)<br>
at org.hl7.fhir.dstu3.utils.ProfileUtilities.generateTable(ProfileUtilities.java:1566)<br>
at org.hl7.fhir.igtools.renderers.StructureDefinitionRenderer.diff(StructureDefinitionRenderer.java:314)<br>
at org.hl7.fhir.igtools.publisher.Publisher.generateOutputsStructureDefinition(Publisher.java:2064)<br>
at org.hl7.fhir.igtools.publisher.Publisher.generateOutputs(Publisher.java:1799)<br>
at org.hl7.fhir.igtools.publisher.Publisher.generate(Publisher.java:1254)<br>
at org.hl7.fhir.igtools.publisher.Publisher.execute(Publisher.java:236)<br>
at org.hl7.fhir.igtools.publisher.Publisher.main(Publisher.java:2194)</p>
<p>Has anyone seen this before?</p>
<p>I just confirmed... the ValueSet is valid against both schema and schematron</p>
<p>Seems to be something related to the binding of the value set in a structure definition</p>
<p>Anyone know the inner-workings of the ig publisher when it comes to value sets?</p>
<p>I see that it connects to grahame's terminology server as one of the starting processes of the publisher</p>
<p>is it trying to retrieve the value set from grahame's server even though I already have it defined (and expanded) in the ig?</p>
<p>Nope.. that can't be it, I just posted the value set to the server no prob... ig publisher still complains</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
